### ISSUE TITLE
files shared with nextcloud have part of the folder name in them

### ISSUE THREAD

**Issue by bahbarnett** [nextcloud/android#6913]
### Steps to reproduce
1. Use a file manager, and 'share' a file with nextcloud
2. See nextcloud 'folder' request, asking you where to put the file
3. Select folder, click OK

### Expected behaviour
- File should upload to the folder

### Actual behaviour
- File tries to upload to the folder, but its name is conflated with the folder name. 

One other thing ... the "conflict resolution" dialog doesn't show the filename, so I never really know what file is in conflict.  It just shows a picture.

Another... this particular issue results in long filenames, so all I can ever see is <part of filename>...<end of filename>.  The only way I could find out precisely what the new filename was, was to try to rename the file.  Hence, the attached 'rename file' dialog.

Basically, I attempted to share (as I've done for years) a file with nextcloud on android.  Everything works fine, except now, it has part of the directory name it in.  -storage-emulated-0- should not be there, only 'database.kdb'.  FYI, I can share fine with everything else, and this worked fine until the most recent update.

![Screenshot_20200908-152949_Nextcloud](https://user-images.githubusercontent.com/2877914/92520337-f7404300-f20a-11ea-8987-f7eaa729a496.jpg)
![Screenshot_20200908-153717_Nextcloud](https://user-images.githubusercontent.com/2877914/92520339-f7404300-f20a-11ea-9630-1eadebb8a15d.jpg)

**Comment by bahbarnett** [comment]
Tried to reproduce a few times, but can't.  I will keep an eye on this, and report back.

**Comment by tobiasKaminsky** [comment]
> Tried to reproduce a few times, but can't. I will keep an eye on this, and report back.

Ok, just re-open once it happens again.
As NC app only uses the info the other app shared with us, I think this was maybe a hiccup on the other side.
